zephyr/src/utils.c

```diff
@@ -452,13 +452,16 @@ int send_udp_data(char *target_ip, int target_port, int packet_count, int packet
         timeout.tv_usec = 0;
     }
     if (is_bridge_created()) {
-        snprintf(ifname, sizeof(ifname), "%s", get_wlans_bridge());
+        ret = snprintf(ifname, sizeof(ifname), "%s", get_wlans_bridge());
```

Comment:
if you move the CHECK_SNPRINTF macro to a common header, you can use it here as well?